DataConnectors/O365 Data/O365APItoAS-Template/host.json

```diff
@@ -2,5 +2,6 @@
   "version": "2.0",
   "managedDependency": {
     "enabled": true
-  }
+  },
```

Comment:
@xyzzykdj : Please update O365APItoAS-Template.zip as well to include the updated host.json

Thanks!!

Reply:
@xyzzykdj: Could you please resolve the review comments?